kyuubi-server/pom.xml

```diff
@@ -133,6 +133,12 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.hudi</groupId>
+            <artifactId>${hudi.name}</artifactId>
```

Comment:
IMO, we do not need to follow `${iceberg.name}`, it can be written here directly.